src/SmartComponents/Systems/EdgeSystemsBanner.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import { useIntl } from 'react-intl';
+import { Alert } from '@patternfly/react-core';
+import { Get } from '../../Utilities/Api';
+import messages from '../../Messages';
+import { INVENTORY_API_BASE } from '../../AppConstants';
+
+const getEdgeSystem = async () =>
+  Get(
+    `${INVENTORY_API_BASE}/hosts?page=1&per_page=1&filter[system_profile][host_type]=edge`
+  );
+
+const EdgeSystemsBanner = () => {
+  const intl = useIntl();
+  const [isLoading, setLoading] = useState(true);
+  const [edgeSystemExist, setEdgeSystemExist] = useState(true);
+
+  useEffect(() => {
+    getEdgeSystem().then((edgeSystem) => {
```

Comment:
As of yesterday, there is AccountStatContext that provides hasEdgeDevices ready to consume. So we can make use of it, instead of making an API request.